src/__test__/utils/upload/processSeuratUpload.test.js

```diff
@@ -233,34 +230,4 @@ describe('processUpload', () => {
       expect(axios.request).not.toHaveBeenCalled();
     });
   });
-
-  it('Should not validate .rds files', async () => {
```

Comment:
Why did you remove this test

Reply:
Oh I had removed it before but then I rolled back the changes I had done that motivated this, so I'll add the test again

Reply:
Done